src/icons/general/IconGlobe2.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg';
+
+type Props = {
+    className?: string,
+    color?: string,
+    height?: number,
+    /** A text-only string describing the icon if it's not purely decorative for accessibility */
+    title?: string | React.Element<any>,
+    width?: number,
+};
+
+const IconGlobe2 = ({ className = '', color = '#8EA6B2', height = 12, title, width = 12 }: Props) => (
```

Comment:
Can we use a more descriptive name? What makes this different from the current globe icon? Also, most new icons are using the `TwoTonedIcon` props, which encourages styling via CSS rather than props. FYI @tjgupta

Reply:
I was following existing examples in the naming, there are multiple which follow this style. Its pretty hard to distinguish the differences in words, but I was given this icon to be used for zones. I originally called it `IconZones` but then saw a globe and thought it would be better for it to be generically named

I can remove the color prop to enforce styling via css if that is the direction we are taking, but don't think this icon meets the two toned criteria